environment.py

```diff
@@ -404,6 +404,9 @@ def env() -> Mapping[str, Optional[str]]:
         # instance configured in `AZUL_TDR_SERVICE_URL`.
         'AZUL_SAM_SERVICE_URL': None,
 
+        # OAuth2 Client ID to be used for authenticating users.
```

Comment:
Needs more meat. Mention that this is shared and refer to the respective README section.